clear-containers-debug.service.in

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Clear Container debug console
+
+[Service]
+Environment=PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+StandardInput=tty
+StandardOutput=tty
+PrivateDevices=yes
+Type=simple
+ExecStart=/usr/bin/bash
```

Comment:
I wonder if this should be `/bin/sh` for portability?

Reply:
@jodh-intel yes. I would prefer the most minimal shell that clear linux provides as a package.

@jcvenegas what are our options? I would rather not have a separate debug image just for this. Or if we end up with a separate image, we should always package and distribute it with the non debug image, so that the user has the confidence that they are using the exact same version of all the image components and be able to flip to the debug image by just modifying the toml config file.

Reply:
@mcastelino @jodh-intel today there is not `busybox` in CL, will need to request a new shell .